src/System.Windows.Forms.Design/tests/UnitTests/System/Windows/Forms/Design/StatusCommandUITests.cs

```diff
@@ -0,0 +1,136 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.ComponentModel;
+using System.ComponentModel.Design;
+using System.Drawing;
+using Moq;
+
+namespace System.Windows.Forms.Design.Tests;
+
+public class StatusCommandUITests
+{
+    private readonly Mock<IServiceProvider> _mockServiceProvider;
+    private readonly Mock<IMenuCommandService> _mockMenuCommandService;
+    private readonly Mock<MenuCommand> _mockMenuCommand;
+    private readonly StatusCommandUI _statusCommandUI;
+
+    public StatusCommandUITests()
+    {
+        _mockServiceProvider = new();
+        _mockMenuCommandService = new();
+        _mockMenuCommand = new(null!, null!);
+
+        _mockServiceProvider
+            .Setup(sp => sp.GetService(typeof(IMenuCommandService)))
+            .Returns(_mockMenuCommandService.Object);
+
+        _mockMenuCommandService
+            .Setup(mcs => mcs.FindCommand(MenuCommands.SetStatusRectangle))
+            .Returns(_mockMenuCommand.Object);
+
+        _statusCommandUI = new(_mockServiceProvider.Object);
+    }
+
+    [Fact]
+    public void SetStatusInformation_WithNullComponent_ShouldNotInvokeCommand()
+    {
+        _statusCommandUI.SetStatusInformation(null);
```

Comment:
```suggestion
        _statusCommandUI.SetStatusInformation(selectedComponent: null);
```
We are using argument names with literals per the style guide because it helps readability.